src/System.ServiceModel.Duplex/src/System.ServiceModel.Duplex.csproj

```diff
@@ -19,10 +19,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
 
   <ItemGroup>
-    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' != 'netcore50aot'" />
-    <TargetingPackReference Include="System.ServiceModel" Condition="'$(TargetGroup)' == 'net46'" />
-    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" Condition="'$(TargetGroup)' != 'net46'" >
-      <AdditionalProperties>OSGroup=Windows_NT</AdditionalProperties>
+    <ProjectReference Include="$(SourceDir)System.Private.ServiceModel\src\System.Private.ServiceModel.csproj" >
```

Comment:
Usually project references should match the set of properties defined in the builds file for the project they are referencing.  Since S.P.SM doesn't have an `AnyOS` build, but this project is building as `AnyOS`, this project reference would result in an extra build of S.P.SM that we'd never ship.  Instead you could choose one existing build (EG: the windows_nt one) to treat as the reference assembly for S.P.SM.  And example of this is the change @weshaggard made to System.IO: https://github.com/dotnet/corefx/commit/05a42a3841b706e1bba6cbaeedd630baf3a1a394